### PR TITLE
feat: html file added

### PR DESCRIPTION
**Pull request by Brandonz07**
I completed this activity

### CODE REVIEW

**Comment by luisalejandrosalazar** [comment]
Meets activity requirements.

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:6]: Adjust the document tab title

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  <style>
+    h1 {
+      color: blue;
+    }
+    h3 {
+      color: rgb(60, 167, 255);
+    }
+  </style>
+  <body>
+    <h1>React</h1>
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:17]: Here is a title, a link and a central point, please refactor it

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    <p>React is a JavaScript library for building user interfaces.</p>
+    <ul>
+      <li>
+        Declarative: React makes it painless to create interactive UIs. Design
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:21]: The headers in this list are not properly formatted

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    <title>React</title>
+  </head>
+  <!-- <style>
+    h1 {
+      color: blue;
+    }
+    h3 {
+      color: rgb(60, 167, 255);
+    }
+  </style> -->
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:15]: Don't leave comment code. This is considered as a bad practice.
Please deleted

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 5]
```diff
@@ -0,0 +1,180 @@
...
+      <p>
+        We have several examples on the website. Here is the first one to get
+        you started:
+      </p>
+      <p>
+        import { createRoot } from "react-dom/client"; function HelloMessage({
+        name }) { return &lt;div&gt;Hello {name}&lt;/div&gt;; } const root =
+        createRoot(document.getElementById("container"));
+        root.render(&lt;HelloMessage name="Taylor" /&gt;);
+      </p>
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:137]: Use the appropriate html tag for this code

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 6]
```diff
@@ -0,0 +1,180 @@
...
+            >Where to Get Support</a
+          >
+        </li>
+        <li>
+          <a href="https://reactjs.org/docs/how-to-contribute.html"
+            >Contribuiting Guide</a
+          >
+        </li>
+      </ul>
+      <h2>Examples</h2>
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:125]: There is a missing line o f text and a link.

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html [hunk 7]
```diff
@@ -0,0 +1,180 @@
...
+      <hr />
+
+      <p>
+        The main purpose of this repository is to continue evolving React core,
+        making it faster and easier to use. Development of React happens in the
+        open on GitHub, and we are grateful to the community for contributing
+        bugfixes and improvements. Read below to learn how you can take part in
+        improving React.
+      </p>
+      <h3>Code of conduct</h3>
```
Comment (luisalejandrosalazar) [inline src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:158]: Be sure of include all  links shown in the model.

Comment (Brandonz07) [reply src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:154]: I have added all the links

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html (hunk 6 above)
Comment (Brandonz07) [reply src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html:121]: I have added line

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html (hunk 5 above)
Comment (Brandonz07) [reply src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html]: now use the code

File: src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html (hunk 4 above)
Comment (Brandonz07) [reply src/br/html/A2-Markup-a-readme/A2-Markup-a-readme.html]: remove commented code

**Review by luisalejandrosalazar — APPROVED** [review]
Approved

**Pull request merged**